plugins/inputs/system/processes.go

```diff
@@ -70,17 +70,15 @@ func getEmptyFields() map[string]interface{} {
 		"stopped":  int64(0),
 		"running":  int64(0),
 		"sleeping": int64(0),
+		"idle":     int64(0),
 		"total":    int64(0),
 		"unknown":  int64(0),
 	}
 	switch runtime.GOOS {
 	case "freebsd":
-		fields["idle"] = int64(0)
```

Comment:
I think we should leave these as before, and add "idle" to the linux section.  In my opinion better to report 0 than unset.